classes/Translations/Translations.php

```diff
@@ -56,6 +56,13 @@ public function getTranslations()
             'general' => [
                 'save' => $this->module->l('Save', 'translations'),
                 'testModeOn' => $this->module->l('Test mode is turned on', 'translations'),
+                'multiShop' => [
+                    'title' => $this->module->l('Multistore detected', 'translations'),
+                    'subtitle' => $this->module->l('You must configure your shops one by one for this service, but you will be able to use the same accounts.', 'translations'),
+                    'chooseOne' => $this->module->l('Please select the first shop to configure from the list below :', 'translations'),
+                    'group' => 'Group :',
```

Comment:
There's no space before a colon in English:

```suggestion
                    'group' => 'Group:',
```

Reply:
Done